footprints/templates/design/pathmapper.html

```diff
@@ -44,34 +44,43 @@
 
 <div class="container banner">
     <div class="intro">
-    Some short text here as an overview of what PathMapper is, what we think it will do for scholarly research.
+    <h1>Pathmapper is a web-based application for charting the historical journey of Jewish books across the world, and through time.</h1>
+    <p class="more_intro">Using the Footprints database, Pathmapper traces the movements of these literary works on a map. These paths reveal compelling narratives and dissemination patterns of knowledge, faith, and culture in the global Diaspora.</p>
     <a href="https://ccnmtl-footprints-static-prod.s3.amazonaws.com/design/pathmapper.pdf" target="_blank" class="download hidden-print"><span class="glyphicon glyphicon-download-alt" aria-hidden="true"></span> Download PDF</a>
     </div>
 </div>
 
 <div class="container slidebox clearfix odd">
-    <h3 class="visible-xs visible-sm print-block">Slide 1</h3>
     <div class="visual">
-    <img src="{{STATIC_URL}}img/pathmapper-test.png">
+    <h3 class="visible-xs visible-sm print-block">Mapping Footprints</h3>
+    <img src="https://ccnmtl-footprints-static-prod.s3.amazonaws.com/design/pathmapper_map.jpg">
```

Comment:
The whole site is not accessible, alt text is the least of the issue.. I need this page to be up ASAP and do all the non-priority fixes later.